Sources/glTFSceneKit/GLTFConverter.swift

```diff
@@ -0,0 +1,351 @@
+//
+//  GLTFConverter.swift
+//  glTFSceneKit
+//
+//  Created by sergey.novikov on 03/10/2019.
+//
+
+import SceneKit
+import os
+
+public class GLTFConverter: TextureLoaderDelegate {
+    
+    /// Status will be true if `cancel` was call.
+    @objc open private(set) var isCancelled: Bool = false
+    @objc open private(set) var errorMessage: Error?
```

Comment:
Please, make `errorMessage` as internal variable. I was planning to do it, but because  it was extension, kept as available.